src/Database/Manager/PhpcrManager.php

```diff
@@ -162,4 +182,16 @@ private function getExecutor($initialize = false)
 
         return $executor;
     }
+
+    public function getDriver()
+    {
+        return 'phpcr';
+    }
+
+    private function assertContainerIsSet()
```

Comment:
what about making the ``container`` property private in the parent class with a protected getter doing such check ? It would avoid having to remember to call this assertion each time